js/base-tool.js

```diff
@@ -549,6 +549,163 @@ function baseTool () {
 				});
 			},
 		},
+		{
+			name: "Mass-Modify Pages",
+			desc: "Quickly modify multiple pages.",
+			html: `
+				<div id="d20plus-mass-page-modify" title="Better20 - Mass-Modify Pages">
+					<div id="modify-pages-list">
+						<div class="list" style="transform: translateZ(0); max-height: 490px; overflow-y: scroll; overflow-x: hidden; margin-bottom: 10px;"><i>Loading...</i></div>
+					</div>
+					<hr>
+                    <p><label class="ib"><input type="checkbox" class="select-all"> Select All</label></p>
+					<table id="modifications-table" style="border-collapse: separate; border-spacing: 10px;">
+					<thead>
+						<tr>
+							<th>Overwrite</th>
+							<th>Label</th>
+							<th>Input</th>
+						</tr>
+					</thead>
+					<tbody>
+						<!-- I could not make it work, but it should not be a hard fix -->
+						<!--
+						<tr>
+							<td><input type="checkbox" id="apply_gridtype" name="apply_gridtype"></td>
+							<td><label for="gridtype">Grid Type:</label></td>
+							<td><select id="gridtype">
+								<option selected="" value="square">Square</option>
+								<option value="hex">Hex (V)</option>
+								<option value="hexr">Hex (H)</option>
+								<option value="dimetric">Dimetric</option>
+								<option value="isometric">Isometric</option>
+								</select>
+							</td>
+						</tr>
+						-->
```

Comment:
define "could not make it work"? The "Diagonal Type" chunk, below, looks very similar; what was the problem here?

Reply:
I guess I was too tired to realise that I had a typo (gridtype -> grid_type). The following change made it work.
```suggestion
						<tr>
							<td><input type="checkbox" id="apply_grid_type" name="apply_grid_type"></td>
							<td><label for="grid_type">Grid Type:</label></td>
							<td><select id="grid_type">
								<option selected="" value="square">Square</option>
								<option value="hex">Hex (V)</option>
								<option value="hexr">Hex (H)</option>
								<option value="dimetric">Dimetric</option>
								<option value="isometric">Isometric</option>
								</select>
							</td>
						</tr>
```